Automattic/wp-e2e-tests
# Set cookie to hide GDPR banner regardless of location

Fixes https://github.com/Automattic/wp-e2e-tests/issues/1386

This PR sets a cookie on the home page (for signup tests) and login page (for other specs) that prevents the GDPR cookie banner from showing.

This resolves an issue where running the tests from within the EU results in failures due to the cookie banner overlapping page elements used in the tests.

To test:
1. Set your location as in the EU (e.g. with a VPN).
2. Run the full suite of tests with `./run.sh -g`.
3. Confirm that the GDPR cookie banner does not appear and there are no related test failures.
